4-projects/README.md

```diff
@@ -39,21 +94,109 @@ Change the `BRANCH_NAME` from `development` to `non-production` or `production`
+1. Merge changes to production branch.
+   ```
+   git checkout -b production
+   git push origin production
+   ```
+1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
+1. After production has been applied, apply development and non-production.
+1. Merge changes to development branch.
+   ```
+   git checkout -b development
+   git push origin development
+   ```
+1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
+1. Merge changes to non-production branch.
+   ```
+   git checkout -b non-production
+   git push origin non-production
+   ```
+1. Review the apply output in your Master's web UI (You might want to use the option to "Scan Multibranch Pipeline Now" in your Jenkins Master UI).
+1. You can now move to the instructions in the step [4-projects](../4-projects/README.md).
+
+### Deploying with Cloud Build
+
+1. Clone repo.
+   ```
+   gcloud source repos clone gcp-projects --project=YOUR_CLOUD_BUILD_PROJECT_ID
+   ```
+1. Change freshly cloned repo and change to non-master branch.
+   ```
+   git checkout -b plan
+   ```
+1. Copy contents of foundation to new repo.
+   ```
+   cp -RT ../terraform-example-foundation/4-projects/ .
+   ```
+1. Copy cloud build configuration files for terraform.
+   ```
+   cp ../terraform-example-foundation/build/cloudbuild-tf-* .
+   ```
+1. Copy terraform wrapper script to the root of your new repository.
+   ```
+   cp ../terraform-example-foundation/build/tf-wrapper.sh .
+   ```
+1. Ensure wrapper script can be executed.
+   ```
+   chmod 755 ./tf-wrapper.sh
+   ```
 1. Rename `common.auto.example.tfvars` to `common.auto.tfvars` and update the file with values from your environment and bootstrap.
 1. Rename `shared.auto.example.tfvars` to `shared.auto.tfvars` and update the file with values from your environment and bootstrap.
 1. Rename `development.auto.example.tfvars` to `development.auto.tfvars` and update the file with the `perimeter_name` that starts with `sp_d_shared_restricted`.
 1. Rename `non-production.auto.example.tfvars` to `non-production.auto.tfvars` and update the file with the `perimeter_name` that starts with `sp_n_shared_restricted`.
 1. Rename `production.auto.example.tfvars` to `production.auto.tfvars` and update the file with the `perimeter_name` that starts with `sp_p_shared_restricted`.
 1. Rename `access_context.auto.example.tfvars` to `access_context.auto.tfvars` and update the file with the `access_context_manager_policy_id`.
-1. You will need only once to manually plan + apply the `business_unit_1/shared` environment since `development`, `non-production` and `production` depend on it.
+1. You need to manually plan and apply only once the `business_unit_1/shared` environment since `development`, `non-production` and `production` depend on it.
```

Comment:
```suggestion
1. You must manually plan and apply the `business_unit_1/shared` environment (only once) since the `development`, `non-production` and `production` environments depend on it.
```